inference-engine/tests/functional/plugin/cpu/single_layer_tests/group_convolution.cpp

```diff
@@ -3,51 +3,38 @@
 //
 
 #include <single_layer_tests/group_convolution.hpp>
-#include "cpu_test_utils.hpp"
+#include "../test_utils/cpu_test_utils.hpp"
```

Comment:
add ```target_include_directories(${TARGET_NAME} PUBLIC ${CMAKE_CURRENT_SOURCE_DIR})``` in inference-engine/tests/functional/plugin/cpu/CMakeLists.txt to avoid this situation